app/src/main/java/org/listenbrainz/android/service/YimService.kt

```diff
@@ -9,8 +9,8 @@ import javax.inject.Singleton
 interface YimService {
     
     // https://api.listenbrainz.org/1/stats/user/(user_name)/year-in-music/2022
-    @GET("stats/user/{user_name}/year-in-music/2022")
-    suspend fun getYimData(@Path("user_name") username: String): YimPayload
+    @GET("stats/user/{user_name}/year-in-music/{year}")
```

Comment:
I see you created a separate service then why touch this?

Reply:
Initially I had thought of using the same service.. but then I realized it wouldn't be feasible.. so I made another service.. Ill revert this